src/models/transactions/offerCancel.ts

```diff
@@ -0,0 +1,21 @@
+import { ValidationError } from "../../common/errors";
+import { BaseTransaction, verifyBaseTransaction } from "./common";
+
+export interface OfferCancel extends BaseTransaction {
+    TransactionType: "OfferCancel";
+    OfferSequence: number;
+}
+
+/**
+ * Verify the form and type of an OfferCancel at runtime.
+ * 
+ * @param tx - An OfferCancel Transaction
+ * @returns - Void.
```

Comment:
I think it makes more sense to have return type be `boolean` instead of `void`. We can `return true` at the end of this function.

Reply:
I don't think it makes sense to `return true` if there's no path where you could `return false`.

Reply:
Hmm that's true (no pun). I'm used to expecting verify functions returning a `boolean` as a success indication. I'm assuming users will wrap `verify` functions in a try catch and handle whatever error is thrown.